### PR TITLE
packaging: update spec file for latest release

### PR DESCRIPTION
**Pull request by sandrobonazzola**
packaging: update spec file for latest release and rename the rpm to gluster-ansible-roles

@gobindadas

### CODE REVIEW

**Comment by sandrobonazzola** [comment]
@rchikatw @gobindadas @mykaul can you please review?

**Review by gobindadas — COMMENTED** [review]
LGTM

File: gluster-ansible.spec [hunk 1]
```diff
@@ -1,14 +1,14 @@
 %global rolesdir %{_sysconfdir}/ansible/roles/gluster.ansible
 %global docdir %{_datadir}/doc/gluster.ansible
-%global buildnum 5
+%global buildnum 26
```
Comment (gobindadas) [inline gluster-ansible.spec]: it should be 27.

File: gluster-ansible.spec [hunk 2]
```diff
@@ -42,6 +42,9 @@ install -p -m 644 README.md LICENSE %{buildroot}/%{docdir}
 %license LICENSE
 
 %changelog
+* Fri Apr 01 2022 Sandro Bonazzola <sbonazzo@redhat.com> - 1.0.5-26
```
Comment (gobindadas) [inline gluster-ansible.spec:48]: it should be 1.0.5-27

**Review by gobindadas — CHANGES_REQUESTED** [review]

File: gluster-ansible.spec (hunk 1 above)
Comment (sandrobonazzola) [reply gluster-ansible.spec]: Fixed in https://github.com/gluster/gluster-ansible/pull/137/commits/ee1728bc66295af57f673c0c6f9c352ac8186f04

File: gluster-ansible.spec (hunk 2 above)
Comment (sandrobonazzola) [reply gluster-ansible.spec:48]: Fixed in https://github.com/gluster/gluster-ansible/pull/137/commits/ee1728bc66295af57f673c0c6f9c352ac8186f04

**Review by gobindadas — APPROVED** [review]
LGTM

**Pull request merged**